enumerable.rb

```diff
@@ -1,72 +1,110 @@
-module Enumerable
-    def my_each(arr)
-result=[]
-for i in 0..length
-    result.push(yield(arr[i]))
-end
-result
-    end
- 
-    def my_each_with_index(arr)
-        result = []
-        for i in 0..length
-            result.push(yield(arr[i],i))
-        end
-        result
-    end
-
-    def my_select
-        result=[]
-        my_each do |x|
-            output.push(x) if yield x
-        end
-        result
-    end
-
-    def my_all
-        result = true
-        my_each do |x|
-            result = false unless yield x
-        end
-        result
-    end
-
-    def my_any?
-        result = false
-        my_each do |x|
-            result = false if yield x
-        end 
-        result
-    end
-
-    def my_none?
-        result = true
-        my_each do |x|
-            result = false if yield x
-        end
-        result
-    end
-
-    def my_count
-        count = 0
-        my_each do |x|
-            count +=1 if yield x
-        end
-        count
-    end
-
-    def my_map(arr)
-        result = []
-        my_each(arr) do |x|
-            result.push(yield(x))
-        end
-        result
-    end
-
-
-
-
-
-
-
-end
\ No newline at end of file
+module_Enumerable
```

Comment:
* [ ] revert this back to ```module Enumerable```, which mean you wrap everything except ```multiply_els``` in the module.
``` module Enumerable ```

>           all methods except ```multiply_els```

``` end ```